### PR TITLE
add bania hud

### PR DESCRIPTION
**Pull request by HUBM3N**
https://imgur.com/a/ULn1arh

### CODE REVIEW

File: hud-data/bania-hud.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "Bania Hud",
+  "": "a hud i made lolololol",
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:3]:
```suggestion
```

File: hud-data/bania-hud.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  "author": "smh2211",
+  "contributors": [
+    "Hypnotize",
+    "Chippy",
+    "Village Green Preserver",
+    "weareallimaginary"
+  ],
+  "releaseDate": "2024-05-20",
+  "resources": [
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:12]: `resources` is meant to list the names of the images you'd have in `hud-resources/bania-hud`:
chrome_KJ0vaR7y7y

File: hud-data/bania-hud.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "huds.tf discord",
+    "fontforge",
+    "paint.net",
+    "notepad++"
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:16]:
```suggestion
    "bania-hud-banner"
```

File: hud-data/bania-hud.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    "fonts",
+    "menus",
+    "scoreboards",
+    "materials"
+  ],
+  "tags": [
+    "minimal",
+    "box",
+    "black and white",
+    ""
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:28]:
```suggestion
    "black and white"
```

File: hud-data/bania-hud.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    "res": [
+      "4/3",
+      "16/9",
+      "16/10"
+    ],
+    "position": [
+      "center"
+    ],
+    "lang": [
+      "English"
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:63]:
```suggestion
      "en"
```
you use the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) codes for these (presumably at least, even though tf2 has a few other ones used i think)

just change it to en

File: hud-data/bania-hud.json [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    ]
+    "resources": [
+      "bania-hud-banner"
+    ]
```
Comment (rbjaxter) [inline hud-data/bania-hud.json:67]:
```suggestion
```

this now becomes a duplicate once the other one is corrected

File: hud-pages/bania-hud.md
Comment (rbjaxter) [inline hud-pages/bania-hud.md:1]: This is a markdown (.md) file, so you use the markdown language: https://www.markdownguide.org/basic-syntax/

File: hud-pages/bania-hud.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+Box hud inspired by and built on bxhud.
+  
+GitHub:  
+<https://github.com/HUBM3N/Bania-Hud>
```
Comment (rbjaxter) [inline hud-pages/bania-hud.md:4]:
```suggestion
https://github.com/HUBM3N/Bania-Hud  

```

File: hud-pages/bania-hud.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+Box hud inspired by and built on bxhud.
+  
+GitHub:  
+<https://github.com/HUBM3N/Bania-Hud>  
+Gamebanana:  
+<https://gamebanana.com/mods/514784>
```
Comment (rbjaxter) [inline hud-pages/bania-hud.md:6]:
```suggestion
https://gamebanana.com/mods/514784  

```

File: hud-pages/bania-hud.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+Box hud inspired by and built on bxhud.
+  
+GitHub:  
+<https://github.com/HUBM3N/Bania-Hud>  
+Gamebanana:  
+<https://gamebanana.com/mods/514784>  
+Screenshots:  
+<https://github.com/HUBM3N/Bania-Hud>
```
Comment (rbjaxter) [inline hud-pages/bania-hud.md:8]:
```suggestion
https://github.com/HUBM3N/Bania-Hud  

```

File: hud-pages/bania-hud.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+Box hud inspired by and built on bxhud.
+  
+GitHub:  
+<https://github.com/HUBM3N/Bania-Hud>  
+Gamebanana:  
+<https://gamebanana.com/mods/514784>  
+Screenshots:  
+<https://github.com/HUBM3N/Bania-Hud>  
+Steam Profile:  
+<https://steamcommunity.com/profiles/76561198992606708>
```
Comment (rbjaxter) [inline hud-pages/bania-hud.md:10]:
```suggestion
https://steamcommunity.com/profiles/76561198992606708

```

**Review by rbjaxter — CHANGES_REQUESTED** [review]

**Review by mastercoms — DISMISSED** [review]

**Review by mastercoms — APPROVED** [review]

**Pull request merged**